tree.py

```diff
@@ -44,14 +45,14 @@
     NamedTuple,
     cast,
 )
+from typing_extensions import Required, TypedDict
 
-from types import ModuleType
 
 import json
 import re
 
-from collections import OrderedDict
 import abc
+from collections import OrderedDict
```

Comment:
Actually, we should get rid of all OrderedDicts in the codebase and replace them with regular dicts. As of Python 3.7, all Python dicts are ordered, and we now only support 3.7+.

Reply:
Ah interesting, I didn't add that import, I think I just moved it to be above the imports from our packages to make the imports look nicer. We can address the OrderedDict changes in a separate PR.

Reply:
Yes, be a little bit careful about this one - I vaguely remember having had a reason to use OrderedDict.